### PR TITLE
Fix/use airbnb base

### PR DESCRIPTION
**Pull request by adam2k**
Changing to `eslint-config-airbnb-base` so this package can work across all of our JS/TS repos.  We can add the React specific pieces into another library or directly into the UI projects as needed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,11 +23,9 @@
...
   "peerDependencies": {
-    "eslint-config-airbnb": "^18.2.0"
-  },
-  "devDependencies": {
     "babel-eslint": "^10.1.0",
-    "eslint": "^7.7.0",
-    "eslint-config-airbnb": "^18.2.0"
+    "eslint": "^7.15.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-jest": "^24.1.3"
```
Comment (adam2k) [inline package.json:29]: @samblackk @ryanahamilton I figured adding these as peerDependencies would be easier to maintain so where ever we use this package we can insure the same versions.  That way we don't need to install all of them directly in ever repo.

Does this go against best practices?

Comment (adam2k) [reply package.json:29]: I think maybe this should be dependencies.  I was misunderstanding the use-case for peerDependencies.  I'll make an update to this PR.

Comment (samblackk) [reply package.json:29]: These would be dependencies :)

**Comment by samblackk** [comment]
@adam2k I agree, this is the approach that should be taken. The `airbnb-base` transgresses across all js. When we import  into our react apps we can extend it as needed.

**Review by samblackk — APPROVED** [review]

**Pull request merged**